source/frontend/src/features/mapSideBar/property/tabs/propertyDetails/detail/PropertyDetailsTabView.tsx

```diff
@@ -63,19 +64,32 @@ export const PropertyDetailsTabView: React.FunctionComponent<IPropertyDetailsTab
 
   const isVolumetricParcel = stringToBoolean(property?.isVolumetricParcel || '');
 
+  const canEditDetails = (property?: IPropertyDetailsForm) => {
+    if (exists(property) && property.isRetired) {
+      return false;
+    }
+    return true;
+  };
+
   return (
     <StyledSummarySection>
       <LoadingBackdrop show={loading} parentScreen={true} />
       <StyledEditWrapper className="mr-3 my-1">
-        {hasClaim(Claims.PROPERTY_EDIT) && (
+        {hasClaim(Claims.PROPERTY_EDIT) && canEditDetails(property) ? (
```

Comment:
My take is that users without the edit claim should still see nothing rendered and ONLY users with the edit claim would see the tooltip.

Reply:
updated